README.md

```diff
@@ -1,3 +1,45 @@
 # recurly
 
 haskell partial recurly api
+
+### How to develop recurly alongside Smurf locally
```

Comment:
LOVE this addition! Future us will be thanking you for sure 🤩

Reply:
Just a reminder for all @EdutainmentLIVE/engineers - this is a public facing repository, so we should probably change it to private before merging this change.